scripts/conda/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+PACKAGE="lumen"
+
+python -m build -w .
+
+VERSION=$(python -c "import $PACKAGE; print($PACKAGE._version.__version__)")
+export VERSION
+
+# conda config --env --set conda_build.pkg_format 2
```

Comment:
We can enable this after I have checked the output of the build artifact and the latest dev release.